docs/how-to/jupyter-notebook/delete-notebook.rst

```diff
@@ -0,0 +1,65 @@
+Delete a Notebook
+=================
+
+This guide provides instructions on how to remove a Jupyter Notebook from the Data Science Stack (DSS) environment.
+
+Overview
```

Comment:
I would like us to have a note here to tell users that some of the data (stored on the PVCs) will not get lost.

Will depend on the next step we'll agree on https://github.com/canonical/data-science-stack/issues/102. Keeping this comment for visibility and let's sync in parallel on the other issue on what our plan will be for the shared PVC for `v0.1` to then decide what to document here

Reply:
Added a note based on the agreed resolution from v.1. Should we keep this comment ?

Reply:
Looks good!